### PR TITLE
[nan-439] allow no output for script

### PR DESCRIPTION
**Pull request by khaliqgant**
## Describe your changes
If an action has not output it would throw an error in the webapp

## Issue ticket number and link
NAN-439

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: packages/webapp/src/utils/utils.tsx [hunk 1]
```diff
@@ -322,6 +322,9 @@ export const parseInput = (flow: Flow) => {
 };
 
 export function generateResponseModel(models: NangoSyncModel[], output: string, isSync: boolean): Record<string, any> {
```
Comment (bodinsamuel) [inline packages/webapp/src/utils/utils.tsx]:
```suggestion
export function generateResponseModel(models: NangoSyncModel[], output: string | undefined, isSync: boolean): Record<string, any> {
```
The type is incorrect then, and that also means Flow's type is also wrong if `output`(returns) is optional

Comment (khaliqgant) [reply packages/webapp/src/utils/utils.tsx]: Updated https://github.com/NangoHQ/nango/pull/1797/commits/71e47d002478a281ed8b820919d457903efdf529

output is optional already actually https://github.com/NangoHQ/nango/blob/master/packages/webapp/src/types.ts#L219

Comment (bodinsamuel) [reply packages/webapp/src/utils/utils.tsx]: EndpointReference.tsx is using `returns` not `output` (I don't know the difference between those 2 properties)

**Review by bodinsamuel — APPROVED** [review]

File: packages/webapp/src/utils/utils.tsx (hunk 1 above)
Comment (khaliqgant) [reply packages/webapp/src/utils/utils.tsx]: `returns` is the v1 legacy property and `output` is the v2 property.

**Pull request merged**